configs/peer/config.json

```diff
@@ -14,7 +14,7 @@
     "PEER_ID": null,
     "BLOCK_TIME_MS": 1000,
     "TRUSTED_PEERS": null,
-    "COMMIT_TIME_LIMIT_MS": 2000,
+    "COMMIT_TIME_LIMIT_MS": 4000,
```

Comment:
Actually, we should reconsider all of the time limits, not just this one.

Reply:
Well, runtime validators execution is not as fast as execution of builtin ones. So I had to increase this parameter. I'll try to experiment and see if I can make it smaller, because it was a long time ago I updated this

Reply:
Okay, turns out it works normally with `2000`. I think I set `4000` before some optimizations